src/Pages/mockCats.js

```diff
@@ -1,35 +1,33 @@
-const cats = [
-    {
-      id: 1,
-      name: "Hiro",
-      age: 25,
-      enjoys: "sunshine and warm spots",
-      image:
-        "https://images.unsplash.com/photo-1543852786-1cf6624b9987?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=687&q=80"
-    },
-    {
-      id: 2,
-      name: "Soza",
-      age: 10,
-      enjoys: "being queen of the dogs",
-      image:
-        "https://images.unsplash.com/photo-1533743983669-94fa5c4338ec?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1092&q=80"
-    },
-    {
-      id: 3,
-      name: "Andrew",
-      age: 7,
-      enjoys: "getting all the attention",
-      image:
-        "https://images.unsplash.com/photo-1592194996308-7b43878e84a6?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=687&q=80"
-    },
-    {
-        id: 4,
-        name: "McKenley",
-        age: 12,
-        enjoys: "Watching the Chiefs game",
-        image:
-          "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcT6LCjQoBWtmaSh1gbyZEcZ_m0znTECb2iI9Q&usqp=CAU"
-    }
-  ]
-  export default cats
\ No newline at end of file
+// const cats = [
```

Comment:
Why is your mockCats commented out? I would recommend keeping them, even if you are fetching from the backend for testing purposes.